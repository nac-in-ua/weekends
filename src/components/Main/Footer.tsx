function Footer() {
  return (
    <footer className="mb-2 flex flex-col items-center justify-center text-center text-base font-thin">
      <a className="flex" href="mailto: nac.in.ua@gmail.com">
        nac.in.ua
      </a>
    </footer>
  )
}

export default Footer
