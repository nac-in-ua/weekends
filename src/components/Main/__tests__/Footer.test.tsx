import Footer from '../Footer'
import { render, screen } from '@testing-library/react'

describe('Footer', () => {
  it('should render link with text', () => {
    render(<Footer />)
    const footerLink = screen.getByRole('link')
    expect(footerLink).toBeInTheDocument()
    expect(footerLink).toMatchSnapshot()
  })
  it('should has attributes', () => {
    render(<Footer />)
    const footerLink = screen.getByRole('link')
    expect(footerLink).toHaveAttribute('href', 'mailto: nac.in.ua@gmail.com')
  })
})
