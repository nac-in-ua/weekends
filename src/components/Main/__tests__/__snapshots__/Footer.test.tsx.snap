// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render link with text 1`] = `
<a
  class="flex"
  href="mailto: nac.in.ua@gmail.com"
>
  nac.in.ua
</a>
`;
